native/cocos/scene/RenderWindow.cpp

```diff
@@ -56,7 +56,17 @@ RenderWindow::RenderWindow()
   _colorName("Color" + std::to_string(_renderWindowId)),
   _depthStencilName("DepthStencil" + std::to_string(_renderWindowId)) {}
 
-RenderWindow::~RenderWindow() = default;
+RenderWindow::~RenderWindow() {
+    // NOTE: destroy needs to be invoked in the destructor of RenderWindow to avoid wild pointer issues in gfx backend code.
+    // RenderWindow owns `_frameBuffer` and `_colorTextures`, `_frameBuffer` should be released before `_colorTextures`
+    // since gfx::Framebuffer keeps a weak pointer of `_colorTextures` and there is code :
+    // GLES3Device::getInstance()->framebufferHub()->disengage(colorTexture->gpuTexture(), _gpuFBO);
```

Comment:
Why not FrameBuffer check validation of colortexture before using it as it is just a weak pointer.

Reply:
colorTexture is released in RenderWindow and Framebuffer owns a weak pointer which is a wild pointer, we could not check whether a pointer is wild.

Reply:
Ok, i thought "weak pointer" is a type of smart pointer. It is a raw pointer.